soccer/planning/RRTPlanner.cpp

```diff
@@ -195,6 +195,28 @@ float getTime(InterpolatedPath& path, int index,
         motionConstraints.maxAcceleration, startSpeed, endSpeed);
 }
 
+std::unique_ptr<Planning::InterpolatedPath> RRTPlanner::generateVelocityPath(
```

Comment:
Well, its a working implementation which just uses the current bezierPath method. (And I branched off this branch for my velocity stuff)